build/constants.yaml

```diff
@@ -30,7 +30,7 @@
       file-name-prefix: __
 - name: dot-net-core-run-time-versions
   constants:
-    dot-net-core-runtime-base-tag: 20200331.2
+    dot-net-core-runtime-base-tag: 20200424.2
```

Comment:
Same here, other base tags in this `constants.yaml` not updated

Reply:
This is because I added this library only in build and .NET Core runtime images. We typically update _all_ the runtime base tags of platforms unless it is required to do so.